docs/upgrading.md

```diff
@@ -72,15 +72,15 @@ Minification using SWC is an opt-in flag to ensure it can be tested against more
 
 ### Improvements to styled-jsx CSS parsing
 
-On top of the Rust-based compiler we've implemented a new CSS parser based on the CSS parser that was used for the styled-jsx Babel transform. This new parser has improved handling of CSS and now errors when invalid CSS is used that would previously slip through and cause unexpected behavior.
+On top of the Rust-based compiler we've implemented a new CSS parser based on the CSS parser that was used for the styled-jsx Babel transform. This new parser has improved the handling of CSS and now errors when invalid CSS is used that would previously slip through and cause unexpected behavior.
 
-Because of this change invalid CSS will throw an error during development and `next build`. This change only affects styled-jsx usage.
+Because of this change, invalid CSS will throw an error during development and the `next build`. This change only affects styled-jsx usage.
```

Comment:
```suggestion
Because of this change, invalid CSS will throw an error during development and `next build`. This change only affects styled-jsx usage.
```